application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -155,38 +157,40 @@ private boolean isAuthorOnCooldown(long userId) {
             .isPresent();
     }
 
-    private MessageCreateAction createModMessage(String reportReason, User author,
-            String reportedMessage, String reportedMessageUrl, String reportedMessageTimestamp,
-            TextChannel modMailAuditLog) {
+    private MessageCreateAction createModMessage(String reportReason, String authorName,
+            String authorAvatarURL, String authorID, String reportedMessage,
+            Instant reportedMessageTimestamp, TextChannel modMailAuditLog) {
         MessageEmbed reportedMessageEmbed = new EmbedBuilder().setTitle("Report")
-            .setDescription("""
-                    Reported Message: **%s**
-                    [Reported Message URL](%s)""".formatted(reportedMessage, reportedMessageUrl))
-            .setAuthor(author.getName(), null, author.getAvatarUrl())
-            .setTimestamp(Instant.parse(reportedMessageTimestamp))
+            .setDescription("Reported Message: **%s**".formatted(
```

Comment:
please get rid of the `"reported message: **...**"`, just `.setDescription(MessageUtils.abbreviate(reportedMessage, ...))`

then it looks more natural.